awx/ui/README.md

```diff
@@ -57,22 +57,13 @@ npm --prefix awx/ui run e2e
 
 ## Adding dependencies
 ```shell
-# add a development or build dependency
```

Comment:
This just makes the "Adding Exact Dependencies" into the regular "Adding Dependencies" section and removes the old one.